tools/djxl_fuzzer.cc

```diff
@@ -334,6 +337,22 @@ bool DecodeJpegXl(const uint8_t* jxl, size_t size, size_t max_pixels,
         seen_need_image_out = true;
       }
 
+      if (info.num_extra_channels > 0) {
+        size_t ec_index = info.num_extra_channels - 1;
```

Comment:
Probably a good idea to make the requested index a random number % (info.num_extra_channels). Or better % (info.num_extra_channles+1) and use one value to not request extra channels even if there are extra channels.

Reply:
Done